### PR TITLE
Update yml formating for extra javascript & css

### PR DESCRIPTION
**Pull request by SteveAzz**
Fix issue #6123

### CODE REVIEW

File: admin/code/LeftAndMain.php [hunk 1]
```diff
@@ -212,8 +212,8 @@ class LeftAndMain extends Controller implements PermissionProvider {
 	 * <code>
 	 * LeftAndMain:
 	 *   extra_requirements_css:
-	 *     mysite/css/mystyle.css:
-	 *       media: screen
+	 *     - mysite/css/mystyle.css:
+	 *       	media: screen
```
Comment (stevie-mayhew) [inline admin/code/LeftAndMain.php]: YML doesn't handle tabs well, this should be 2 space indented

Comment (SteveAzz) [inline admin/code/LeftAndMain.php]: Should be fixed now.

**Comment by stevie-mayhew** [comment]
Thanks @SteveAzz

**Comment by tractorcow** [comment]
This changed all indentation in LeftAndMain.php to spaces. We will do this once we apply psr-2, but we haven't yet changed the standard setting.

**Comment by tractorcow** [comment]
PR at https://github.com/silverstripe/silverstripe-framework/pull/6162

**Comment by SteveAzz** [comment]
@tractorcow Do you have a PR already set for PSR-2? Are you planing to you something like [php-cs-fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) and then maybe implement something like [styleci](https://styleci.io/)

**Comment by tractorcow** [comment]
@SteveAzz see https://github.com/silverstripe/silverstripe-framework/issues/6096

**Pull request merged**